x-pack/platform/packages/shared/kbn-ai-assistant/src/chat/welcome_message_knowledge_base.tsx

```diff
@@ -100,69 +117,72 @@ export function WelcomeMessageKnowledgeBase({
         </>
       ) : null}
 
-      {connectors.connectors?.length ? (
```

Comment:
moved check to parent.  what case would we want to mount this component before setting up a connector?  can't think of one.